internal/components/fluentd/run_script.go

```diff
@@ -132,6 +132,46 @@ do
     fi
 done
 
+FILE="/var/lib/fluentd/pos/journal_pos.json"
+
+    echo "$FILE exists."
+
+if test -f "$FILE"; then
+    echo "$FILE exists."
+    ruby -v /etc/fluent/configs.d/user/cleanInValidJson.rb  2> /tmp/error-json-parser.log
```

Comment:
it'd be much more UNIX-like if the check simply returned 0 for valid and 1 for invalid JSON

Reply:
and the JSON file to check to be a parameter

Reply:
and cleanInValidJson.rb can be a here doc that can be fed to ruby on stdin